src/messages.tsx

```diff
@@ -0,0 +1,20 @@
+export const NO_CONTEXTS_SAVED = 'No contexts saved yet, run asyncapi --help to know more';
+
+export const CONTEXT_NOT_FOUND = 'This context key does not exists.';
+
+export const KEY_NOT_FOUND = 'Key not found';
+
+export const DELETING_CURRENT_CONTEXT = 'You are tyring to delete a context that is currently in use.';
```

Comment:
spelling error and also context name should be in the message

`Context {CONTEXT_NAME} is set as current and cannot be removed`

Reply:
This one is currently not in use. I created it for a warning case if the user is deleting any context that is set as current but didn't end up using it. Should I remove this?

Reply:
yup